.gitignore

```diff
@@ -9,5 +9,3 @@ coverage/*
 
 nbproject/*
 dist/
-chartjs-plugin-zoom.js
```

Comment:
you should probably leave these and add the esm files as well for packaging. see my other comment below

Reply:
If I'm understanding correctly, we don't need these .gitignore lines or the gulpfile `copyDistFilesTask`.

The previous configuration invoked Rollup to put the files under `dist`, then it called `copyDistFilesTask` to copy the files out of `dist` into the project root (which is why the files are listed in .gitignore here). But package.json references the copies under `dist` directory, and the gulpfile's `packageTask` references the copies under `dist` (via `outDir`). I can't tell that the copies in the project root are used at all, so I removed them.

If I've missed something, please let me know.

Reply:
`gulp package` looks like it looks for files in the root directory: https://github.com/chartjs/chartjs-plugin-zoom/blob/8194ad68db5d197c7eea539fc2a4b4bd4ef8c082/gulpfile.js#L64

Alternatively, you might be able to update `gulp package` to look in `dist`

Reply:
If I'm understanding Gulp correctly, it treats `outDir` as the [glob base](https://gulpjs.com/docs/en/api/concepts#glob-base) for `outDir + '*.js'`, so the files from `dist` are included at the top level of the resulting zip file. Contrast with the explicit `base` parameter for the `samples` glob; those are included in a `samples` subdirectory within the zip file:

https://github.com/chartjs/chartjs-plugin-zoom/blob/8194ad68db5d197c7eea539fc2a4b4bd4ef8c082/gulpfile.js#L68

I tested by running `npm run build; npx gulp package` in a fresh copy of my branch, and it created a `dist/chartjs-plugin-zoom.zip` file with contents similar to the releases on GitHub, so I think things are working.

Reply:
Ah, we might need to update the comments in the `packageTask` as they threw me off and look like they might be outdated:
> // gather "regular" files landing in the package root
> // since we moved the dist files one folder up (package root), we need to rewrite
> // samples src="../dist/ to src="../ and then copy them in the /samples directory.

Reply:
Yes, I found that confusing too. I just pushed some comments to try and clarify; feedback welcome.